transcom/mymove
# MB-16229 remove unneeded TIO permissions

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16229)

## Summary

This is a very simple PR that removes the specified permissions from the TIO. 
NOTE that as discussed in slack, TOOs will retain their permission to read payment requests and a [different PR ](https://github.com/transcom/mymove/pull/10602/files#diff-eeb0b6ca29a211250108bba5b89b0feaad731dc714d65ce969e2dc8c763d0770L35)has revoked the TOOs ability to update the payment service item status. 

## Verification Steps for the Author

These are to be checked by the author.


## Test Steps

- Sign into the office app as `tio_role@office.mil` 
- Select a payment request and navigate to the 'Move Task Order' tab
- Verify that you cannot see the option to click on links for requesting a cancellation, a reweigh or a diversion
- If you sign in as `too_role@office.mil` and view the same move's "Move Task Order" page you should be able see the links for requesting a cancellation, reweigh or a diversion. 

## Screenshots
View as a TIO (revoked permissions)

<img width="1024" alt="Screenshot 2023-06-12 at 2 32 44 PM" src="https://github.com/transcom/mymove/assets/67110378/2d5bf5f7-3d3a-4839-b986-646b85954500">

View as a TOO (enabled permissions)

<img width="985" alt="Screenshot 2023-06-12 at 2 33 16 PM" src="https://github.com/transcom/mymove/assets/67110378/7b6b70c2-5e6a-4336-a94a-8763c0c532a4">
